syntax = "proto3";

package greet;

service Greeter {
  rpc SayHello (HelloRequest) returns (HelloReply) {}
}

message HelloRequest {
  string client_node_ip = 1;
}

message HelloReply {
    string message = 1;
    string client_node_coordinates = 2;
    string server_node_coordinates = 3;
    string additional_connections = 4;
}